src/api/app/views/webui2/shared/_download_repository_link.html.haml

```diff
@@ -0,0 +1,6 @@
+- if @configuration['download_url']
+  -# haml-lint: disable  InstanceVariables
```

Comment:
You can pass `@configuration` in the locals when calling this partial and remove this line.

Reply:
i will write it differently, just call ```render``` if the ```@configuration['download_url']``` is there.